### DISCUSSION TITLE
How to draw overlay polygons without "hit and miss"

### DISCUSSION THREAD

**Discussion by kamazeuci** [mistic100/Photo-Sphere-Viewer#1435] [General]
Hi! I want to draw something like a map of borders on the sphere. So I plan to use the overlay plugin. But I do not know the exact coordinates of the polygons, what is the best process in order to draw this map and then somehow import coordinates into photo sphere viewer?

You could think of something like this but with more polygons
https://virtual360.cl/2023/02/03/1454/

**Comment by mistic100** [comment]
I think polygon markers are more adapted to your need https://photo-sphere-viewer.js.org/plugins/markers.html#polygon

Here is a small snipped to get coordinates of where you click.

```js
viewer.addEventListener('click', ({ data }) => {
    console.log(data.yaw, data.pitch);
});
```
